tools/plugin_checker.py

```diff
@@ -0,0 +1,188 @@
+#! /usr/bin/env python
+
+import os
+import json
+import requests
+import subprocess
+import yaml
+import argparse
+
+from packaging import version
+from pathlib import Path
+
+DESC_TEXT = "This is a small script which clones a Che plugin registry GitHub repository, \
+    checks the versions of all the plugins, and determines which ones need to be updated. \
+    be sure to fill out TOKEN, REPO_OWNER, REPO_NAME, and REPO_LABELS before running."
+
+# Add arguments
+parser = argparse.ArgumentParser(description = DESC_TEXT)
+parser.add_argument("-f", "--file", help="file issues against the GitHub repo", action="store_true")
+parser.add_argument("-s", "--silent", help="run this script silently without any report details", action="store_true")
+args = parser.parse_args()
+
+
+# NOTE: edit the fields below to customize where and how the script files issues
+# Access token generated by GitHub
+TOKEN = ''
```

Comment:
could it work with environnent variable instead of editing the script ?

Reply:
Yes, I'll investigate it.